__tests__/app.test.js

```diff
@@ -106,6 +114,32 @@ describe("api/articles", () => {
         });
     });
   });
+  describe("GET api/articles", () => {
+    test("Status 200: responds with an array of article objects", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(13);
+          expect(body).toBeSortedBy("created_at");
+          const article_id1 = body.find((article) => article.article_id === 1);
+          expect(article_id1).toHaveProperty("comment_count", "11");
```

Comment:
We shouldn't need these 2 lines of code. It's just checking that the first article has a comment count of 11. Once you've refactored the toBeSortedBy expect statement, this won't be necessary 👍